website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx

```diff
@@ -409,14 +409,26 @@ curl \
   https://app.terraform.io/api/v2/tasks/task-7oD7doVTQdAFnMLV
 ```
 
-## Attach a Run Task to a Workspace
+## Associate a Run Task to a Workspace
 
 `POST /workspaces/:workspace_id/tasks`
 
 | Parameter       | Description              |
 | --------------- | ------------------------ |
 | `:workspace_id` | The ID of the workspace. |
 
+This endpoint associates an existing run task to a specific workspace.
+
+This involves setting the run task enforcement level, which determines whether the run task blocks runs from completing.
+
+- Advisory run tasks can not block a run from completing. If the task fails, the run will proceed with a warning.
+
+- Mandatory run tasks block a run from completing. If the task fails (including a timeout or unexpected remote error condition), the run stops with an error.
+
+You may also configure the run task to begin during a specific [run stage](/cloud-docs/run/states). Run tasks are configured to use the [Post-Plan Stage](/cloud-docs/run/states#5-the-post-plan-stage) by default.
+
+-> **Note:** Configuring the [run stage](/cloud-docs/run/states) for a run task is in beta
```

Comment:
P.S. I really like how you handled this here. Gives context while still leaving room for more future stages - awesome job!

```suggestion
-> **Note:** Configuring the [run stage](/cloud-docs/run/states) for a run task is in beta.
```